cpu/sam0_common/periph/rtc_rtt.c

```diff
@@ -320,6 +351,7 @@ void rtc_init(void)
 
     /* enable overflow interrupt */
     RTC->MODE2.INTENSET.reg = RTC_MODE2_INTENSET_OVF;
+    _pm_block(&_pm_overflow);
```

Comment:
I think instead we can do
```suggestion
    if (RTC->MODE2.INTFLAG.bit.OVF) {
        RTC->MODE2.INTFLAG.reg = RTC_MODE2_INTFLAG_OVF;
        reference_year += 64;
    }
```

Reply:
Huh? Shouldn't the overflow be handled in the overflow ISR handler? Actually extending the reference year blocks BACKUP mode, yes. I don't if this is a widely used feature. I'd be surprised (and scared!) if the same IoT device are around in 64 years :D

Reply:
It might not be widely used, but there is no way to turn it off - so BACKUP would always be blocked.

Reply:
After having some hours of sleep: You're checking if the OVF happend during BACKUP mode, right?

Reply:
Yes that's the idea, I didn't try it but if the device doesn't sleep for longer than 64 years at a time it should be fine.

Reply:
Hmm, I'd rather remove that overflow feature. It will fire in 2084 considering `RIOT_EPOCH` is 2020. Or am I wrong?

Reply:
That would also save a few bytes of RAM as we can keep the year offset `const`

Reply:
See my recent changes.

ROM sizes for `tests/periph_rtc`:

```
# Current master
make -C tests/periph_rtc BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  13160	    180	   2380	  15720	   3d68	/home/jue/Projects/ssv/RIOT/tests/periph_rtc/bin/samr30-xpro/tests_periph_rtc.elf

# This PR with SAM0_RTCRTT_PM_BLOCK undefined (i.e. no pm in this driver)
make -C tests/periph_rtc BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  13060	    176	   2380	  15616	   3d00	/home/jue/Projects/ssv/RIOT/tests/periph_rtc/bin/samr30-xpro/tests_periph_rtc.elf

# This PR with SAM0_RTCRTT_PM_BLOCK defined
make -C tests/periph_rtc BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  13148	    176	   2384	  15708	   3d5c	/home/jue/Projects/ssv/RIOT/tests/periph_rtc/bin/samr30-xpro/tests_periph_rtc.elf
```

ROM sizes for `tests/periph_rtt`:

```
# Current master
make -C tests/periph_rtt BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  11912	    132	   2368	  14412	   384c	/home/jue/Projects/ssv/RIOT/tests/periph_rtt/bin/samr30-xpro/tests_periph_rtt.elf

# This PR with SAM0_RTCRTT_PM_BLOCK undefined (i.e. no pm in this driver)
make -C tests/periph_rtt BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  11912	    132	   2368	  14412	   384c	/home/jue/Projects/ssv/RIOT/tests/periph_rtt/bin/samr30-xpro/tests_periph_rtt.elf


# This PR with SAM0_RTCRTT_PM_BLOCK defined
make -C tests/periph_rtt BOARD=samr30-xpro LTO=1 all
   text	   data	    bss	    dec	    hex	filename
  12000	    132	   2372	  14504	   38a8	/home/jue/Projects/ssv/RIOT/tests/periph_rtt/bin/samr30-xpro/tests_periph_rtt.elf
```